Resources/Prototypes/_RMC14/Roles/Loadouts/Items/RoleSpecific/chief_mp.yml

```
@@ -0,0 +1,15 @@
+# CMP
+- type: loadout
+  id: CMPPeakedCapRMC
+  cost: 3 # same price as headwear
```

We want to make this only cost 1 point at least for now, can change the berets to 1 too in another PR
```suggestion
  cost: 1
```